install.sh

```diff
@@ -444,6 +444,10 @@ do_install() {
 						# also install the latest version of the "docker scan" cli-plugin (only supported on x86 currently)
 						pkgs="$pkgs docker-scan-plugin"
 				fi
+				# TODO(thaJeztah) remove the $CHANNEL check once 22.06 and docker-buildx-plugin is published to the "stable" channel
+				if [ "$CHANNEL" = "test" ] && version_gte "22.06"; then
```

Comment:
Version wouldn't ever be greater than 22.06 in the stable channel yet anyhow, right?  So this is probably safe to remove the `CHANNEL` test from and merge? :pray:

Reply:
So.. lol, it was late when I originally wrote this patch, and I wasn't sure what `version_gte` did if _no version_ was specified explicitly (as "no version" means "install latest"), so I wanted to safeguard against that (Then decided not to take the risk either way, and leave it in draft until later)

I should have a look at that, and if that's not the case, I can remove it.

Reply:
Did a quick try to see what happens if we remove the `$CHANNEL` check, but it's indeed written with the assumption that "if no version is specified, then `version_gte` is always `true`";

```console
curl -fsSL https://raw.githubusercontent.com/thaJeztah/docker-install/add_buildx_package/install.sh -o get-docker.sh

sed -i 's/\[ "$CHANNEL" = "test" ] && //g' ./get-docker.sh 
```

```console
sh ./get-docker.sh

# Executing docker install script, commit:
+ sh -c apt-get update -qq >/dev/null
+ sh -c DEBIAN_FRONTEND=noninteractive apt-get install -y -qq apt-transport-https ca-certificates curl >/dev/null
+ sh -c mkdir -p /etc/apt/keyrings && chmod -R 0755 /etc/apt/keyrings
+ sh -c curl -fsSL "https://download.docker.com/linux/ubuntu/gpg" | gpg --dearmor --yes -o /etc/apt/keyrings/docker.gpg
+ sh -c chmod a+r /etc/apt/keyrings/docker.gpg
+ sh -c echo "deb [arch=amd64 signed-by=/etc/apt/keyrings/docker.gpg] https://download.docker.com/linux/ubuntu jammy stable" > /etc/apt/sources.list.d/docker.list
+ sh -c apt-get update -qq >/dev/null
+ sh -c DEBIAN_FRONTEND=noninteractive apt-get install -y -qq --no-install-recommends docker-ce docker-ce-cli containerd.io docker-compose-plugin docker-scan-plugin docker-buildx-plugin >/dev/null
E: Unable to locate package docker-buildx-plugin
```

We could change that to have the script actually resolve the latest version, so get it from, e.g.;

```console
apt-cache policy 'docker-ce' | grep 'Candidate:' | cut -d' ' -f4
5:20.10.17~3-0~ubuntu-jammy
```

But I guess that would make the script a bit too complicated, just for this purpose.